libraries/push/impl/src/main/res/xml/notifications_provider_paths.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <cache-path
+        name="downloads"
```

Comment:
Should we use `images` or `pictures` instead, so we don't mix them up with manually downloaded files? I know these would be downloaded somewhere else, but there's still the chance of it happening.

Reply:
This is the path for the `fileprovider` (see the example above: `content://io.element.android.x.debug.notifications.fileprovider/downloads/temp/notif/matrix.org/XGItzSDOnSyXjYtOPfiKexDJ`), and I understand this is like virtual path, not really the location where the file is stored.